build.yaml

```diff
@@ -2549,7 +2564,7 @@ steps:
       namespace:
         valueFrom: default_ns.name
       mountPath: /ssl-config
-   timeout: 2400
+   timeout: 5400
```

Comment:
This seems crazy to me that we can have jobs that need 1.5 hours! Can you explain this to me again?

Reply:
I just picked a big number. It took more than 2400 seconds to rebuild every image from scratch. The base image, for example, took 18 minutes. Normally it takes 9 minutes. The context is that the CI-under-test has to use the batch-under-test which makes build jobs use 0.250 CPU. The machines are also tiny, so their bandwidth is lower.